src/components/ListingTabs/ListingTabs.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import './ListingTabs.scss';
+import Tab from '../UI/Tab/Tab';
+import tabsInfoObj from './listingTabsObject';
+import ListingTabCallout from './ListingTabCallout';
+
+const ListingTabs = (props) => {
+  const [tabsInfo] = useState(tabsInfoObj);
+
+  return (
+    <div className={`listing-tabs ${props.className ? props.className : ''}`}>
+      {tabsInfo?.map((tab, i) => (
+        <Tab
+          className="listing-tabs__item"
+          title={tab.title}
+          isActive={i === 0 ? true : false}
```

Comment:
It might make sense to name this prop something like isActiveDefault since it's only used for initializing the state of the tab.